.gitmodules

```diff
@@ -8,5 +8,5 @@
 	branch = master
 [submodule "ccpp/physics"]
 	path = ccpp/physics
-	url =  https://github.com/NCAR/ccpp-physics.git 
```

Comment:
Thanks for reverting `.gitmodules` to where it was originally (the new version in this PR is how it should be, no need to add `.git` and also good to remove the unnecessary double whitespace and the trailing whitespace below.